config/envoyconfig/route_configurations.go

```diff
@@ -30,36 +32,63 @@ func (b *Builder) BuildRouteConfigurations(
 	return routeConfigurations, nil
 }
 
-func (b *Builder) buildMainRouteConfiguration(
-	_ context.Context,
-	cfg *config.Config,
-) (*envoy_config_route_v3.RouteConfiguration, error) {
-	authorizeURLs, err := cfg.Options.GetInternalAuthorizeURLs()
+func (b *Builder) buildRouteConfiguration(name string, virtualHosts []*envoy_config_route_v3.VirtualHost) (*envoy_config_route_v3.RouteConfiguration, error) {
+	return &envoy_config_route_v3.RouteConfiguration{
+		Name:         name,
+		VirtualHosts: virtualHosts,
+		// disable cluster validation since the order of LDS/CDS updates isn't guaranteed
+		ValidateClusters: &wrapperspb.BoolValue{Value: false},
+	}, nil
+}
+
+func (b *Builder) buildMainRouteConfiguration() (*envoy_config_route_v3.RouteConfiguration, error) {
```

Comment:
Is there some way to break this method up? I'm having a hard time following the catch-all loop